poc_iot_verifier/src/runner.rs

```diff
@@ -50,20 +49,17 @@ impl Runner {
             pool,
             settings: settings.clone(),
             // follower_cache,
-            density_queries: None,
```

Comment:
I wanted to the verifier to use an `impl HexDensityMap` and if I put that in a struct, it has to a trait object which enforces dynamic dispatch on the method calls, and rather than jump through those hoops, I just passed it in function arguments which doesn't require any of that.